spotbox/spotboxgui.py

```diff
@@ -156,6 +188,7 @@ def _loadspot(self, spottext, spotnumber):
             spottext.set(self._menus.filesubjecttoload)
             self.countdownarray[spotnumber].loadcountdown(
                 self._menus.filetimetoload)
+            self.spot_durations[spotnumber] = self._menus.filetimetoload
```

Comment:
This is good, I'm glad that the play queue is guaranteed only to contain spots that are explicitly set (as opposed to implicitly playing from all registers, such as iTunes playlists)